metadata.in.json

```diff
@@ -1,12 +1,12 @@
 {
     "id": "yandex_music", 
     "name": "Yandex Music", 
-    "maintainer_name": "Jiří Janoušek", 
-    "maintainer_link": "https://github.com/fenryxo", 
+    "maintainer_name": "Aleksey Zhidkov", 
+    "maintainer_link": "https://github.com/aleksey-zhidkov", 
```

Comment:
404 Not found. Have you renamed your account?

Reply:
Yes, my bad, will fix it